From 6c69c8a5b2715bebd7c3727c1af78828863ccd4e Mon Sep 17 00:00:00 2001
From: Stefan Wahren <stefan.wahren@i2se.com>
Date: Sun, 29 Oct 2017 12:49:05 +0100
Subject: [PATCH 291/454] ARM: dts: bcm283x: Fix DTC warnings about missing
 phy-cells

commit 014d6da6cb2525d7f48fb08c705cb130cc7b5f4a upstream.

This patch fixes the DTC warnings about missing property #phy-cells.

Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>
Signed-off-by: Eric Anholt <eric@anholt.net>
Reviewed-by: Eric Anholt <eric@anholt.net>
---
 arch/arm/boot/dts/bcm283x.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/bcm283x.dtsi
+++ b/arch/arm/boot/dts/bcm283x.dtsi
@@ -640,5 +640,6 @@
 
 	usbphy: phy {
 		compatible = "usb-nop-xceiv";
+		#phy-cells = <0>;
 	};
 };
